File: app/src/main/java/ru/anlim/rmatch/logic/PicaccoHelper.java
package ru.anlim.rmatch.logic;

import android.widget.ImageView;
import android.widget.MediaController;
import android.widget.Toast;

import com.squareup.picasso.MemoryPolicy;
import com.squareup.picasso.NetworkPolicy;
import com.squareup.picasso.Picasso;

public class PicaccoHelper {

    public void LoadPic(String url, ImageView imageView){

        if (url != null){
            Picasso.get()
                    .load(url.trim())
                    .noFade()
                    .into(imageView);
        }
    }
}

File: app/src/main/java/ru/anlim/rmatch/logic/PicaccoHelper.kt
package ru.anlim.rmatch.logic

import android.widget.ImageView
import com.squareup.picasso.Picasso

class PicaccoHelper {
    fun loadPic(url: String?, imageView: ImageView?) {
        if (url != null) {
            Picasso.get()
                .load(url.trim { it <= ' ' })
                .noFade()
                .into(imageView)
        }
    }
}